### DISCUSSION TITLE
Is there any way to send an mp3 or wav to a project?

### DISCUSSION THREAD

**Discussion by chinmaym505** [Sid72020123/scratchconnect#54] [General]
Hi, this is [@chinmaym](https://scratch.mit.edu/users/chinmaym/) from scratch.
I want to know if it is possible to send an mp3 or wav to a project.
Please let me know if I can do it.

**Comment by Sid72020123** [comment]
No! That is not possible as there is no Scratch API endpoint to do that. I will find a way to do that later...
